test/local_creds_test.go

```diff
@@ -52,15 +73,17 @@ func testE2ESucceed(network, address string) error {
 	go s.Serve(lis)
 
 	var cc *grpc.ClientConn
-	if network == "unix" {
-		cc, err = grpc.Dial("passthrough:///"+address, grpc.WithTransportCredentials(local.NewCredentials()), grpc.WithContextDialer(
+	switch network {
+	case "unix":
+		cc, err = grpc.Dial(address, grpc.WithTransportCredentials(local.NewCredentials()), grpc.WithContextDialer(
 			func(ctx context.Context, addr string) (net.Conn, error) {
 				return net.Dial("unix", address)
 			}))
-	} else {
+	case "tcp":
 		cc, err = grpc.Dial(lis.Addr().String(), grpc.WithTransportCredentials(local.NewCredentials()))
```

Comment:
Can this also dial `address` to be consistent with the above?

Reply:
By dialing `address` it will dial with port 0, which is incorrect. I updated the test to use `lisAddr` in both `unix` and `tcp` cases.